src/App.jsx

```diff
@@ -1,80 +1,16 @@
 import React from 'react';
 import './App.css';
+import Welcome from './components/Welcome/Welcome';
+import Header from './components/Header/Header';
+import Article from './components/Article/Article';
```

Comment:
creating `index.` files would shorten the import statements
```suggestion
import Welcome from './components/Welcome';
import Header from './components/Header';
import Article from './components/Article';
```